qa/tasks/ceph_manager.py

```diff
@@ -1054,6 +1177,9 @@ def choose_action(self):
                  self.config.get('chance_inject_pause_long', 0),)]:
                 actions.append(scenario)
 
+        actions.append((self.test_bluestore_reshard,
```

Comment:
should disable this if osd_objectstore is not bluestore - you can check that with something like ctx.ceph[cluster_name].conf.get('osd', {}).get('osd objectstore', 'bluestore') == 'bluestore'